### PR TITLE
Prepare api security tests for dotnet

### PR DESCRIPTION
**Pull request by anna-git**
## Description

- Adapt tests, as payloads are slightly different in dotnet
-  Adapt dotnet weblog
- Fixes : cookies value can't contain whitespaces, actually pass cookies when building the request

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

## Reviewer checklist

* [ ] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] if any of `build-some-image` label is present
  1. is the image labl have been updated ? 
  2. just before merging, locally build and push the image to hub.docker.com
* [ ] if a scenario is added (or removed), add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
there are dotnet tests failures

**Comment by anna-git** [comment]
> there are dotnet tests failures

fixed in https://github.com/DataDog/system-tests/pull/1755/commits/a3ee2e96df1c6a67cf46c3dd41de64e274e1eada

**Review by christophe-papazian — APPROVED** [review]

File: tests/appsec/api_security/test_schemas.py [hunk 1]
```diff
@@ -48,7 +48,9 @@ def test_request_method(self):
         assert self.request.status_code == 200
         assert schema
         assert isinstance(schema, list)
-        assert contains(schema, [{"accept-encoding": [8], "host": [8], "user-agent": [8]}])
+        for parameter_name in ("accept-encoding", "host", "user-agent"):
```
Comment (cbeauchesne) [inline tests/appsec/api_security/test_schemas.py:51]: We should consider to move this kind of assertion inside schema on day.

**Review by NachoEchevarria — APPROVED** [review]

**Review by robertomonteromiguel — APPROVED** [review]

File: tests/appsec/api_security/test_schemas.py (hunk 1 above)
Comment (anna-git) [reply tests/appsec/api_security/test_schemas.py:51]: will resolve for now, but sounds good, for another PR move to https://github.com/DataDog/system-tests/tree/main/utils/interfaces/schemas et describe schemas in json schemas files.

**Pull request merged**